lib/syskit/task_context.rb

```diff
@@ -955,6 +980,12 @@ def setup_failed!(exception)
         # configured and started.
         event :start do |_context|
             info "starting #{self}"
+
+            if read_only?
+                event(:start).emit
```

Comment:
Nope. You need to just ... return

The start event should be emitted only if the C++ task context is running. update_task_states.rb should take care of that.

Reply:
done